_includes/components/content_blocks/text.html

```diff
@@ -1,5 +1,5 @@
 {% assign block = include.block %}
 
-<h3>{{ block.title }}</h3>
+<!-- <h3>{{ block.title }}</h3> -->
```

Comment:
We need the title as a separate attribute for accordion blocks, so duplicating it in the body just makes it appear twice in accordions. I'm almost certain we could find a good way to specifically target the elements in the room views to do whatever we need.